gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -210,7 +210,6 @@
   public static final String MAXIMUM_JAR_COPY_RETRY_TIMES_KEY = JOB_JAR_FILES_KEY + ".uploading.retry.maximum";
   public static final String USER_DEFINED_STATIC_STAGING_DIR = "user.defined.static.staging.dir";
   public static final String USER_DEFINED_STAGING_DIR_FLAG = "user.defined.staging.dir.flag";
-  public static final String IS_DATASET_STAGING_DIR_USED = "dataset.staging.dir.used";
```

Comment:
Note: need to update the docs/wiki pages after this is merged, to reflect the new name.